src/components/utils/TextEditor.js

```diff
@@ -0,0 +1,196 @@
+                    <ToggleButton
+                        value="italic"
+                        aria-label="italic"
+                        onClick={() => editor.chain().focus().toggleItalic().run()}
+                        disabled={disabled}
+                    >
+                        <FormatItalic fontSize="small" />
+                    </ToggleButton>
+                    <ToggleButton
+                        value="underline"
+                        aria-label="underline"
+                        onClick={() => editor.chain().focus().toggleUnderline().run()}
+                        disabled={disabled}
+                    >
+                        <FormatUnderlined fontSize="small" />
+                    </ToggleButton>
+                </ToggleButtonGroup>
+            </Box>
+            <Box mr={2} mb={1} display="inline">
+                <ToggleButtonGroup size="small" value={formats} aria-label="text lists">
+                    <ToggleButton
+                        value="bulletList"
+                        aria-label="bulletList"
+                        onClick={() => editor.chain().focus().toggleBulletList().run()}
+                        disabled={disabled}
+                    >
+                        <FormatListBulleted fontSize="small" />
+                    </ToggleButton>
+                    <ToggleButton
+                        value="orderedList"
+                        aria-label="orderedList"
+                        onClick={() => editor.chain().focus().toggleOrderedList().run()}
+                        disabled={disabled}
+                    >
+                        <FormatListNumbered fontSize="small" />
+                    </ToggleButton>
+                </ToggleButtonGroup>
+            </Box>
+            <Box display="inline">
+                <ToggleButtonGroup size="small" value={formats} aria-label="text lists">
+                    <ToggleButton
+                        className={classes.headingBtn}
+                        value="heading1"
+                        aria-label="heading1"
+                        onClick={() => editor.chain().focus().toggleHeading({ level: 1 }).run()}
+                        disabled={disabled}
+                    >
+                        <Typography>section</Typography>
+                    </ToggleButton>
+                </ToggleButtonGroup>
+            </Box>
+        </div>
+    );
+};
+
+EditorToolbar.propTypes =  {
+    editor: PropTypes.object,
+    disabled: PropTypes.bool,
+};
+
+const TextEditor = ({ content, onChangeDescription, onChangeDescriptionText, error, helperText: additionalHelperText, disabled }) => {
+    const editor = useEditor({
+        extensions: [
+            StarterKit,
+            Underline,
+            Placeholder.configure({
+                placeholder: "Write a description for this offer. You can specify goals, project and daily work details, etc.",
+            }),
+            Heading.configure({
```

Comment:
Weren't we supposed to tweak the Heading plugin in order to not have huge titles in the offer descriptions?